docker/Dockerfile.build-server

```diff
@@ -0,0 +1,17 @@
+# How to build this image:
```

Comment:
I added a PowerShell module to the build server to deal with YAML files.  This will help us to dynamically create docker-compose files.